demisto_sdk/commands/content_graph/commands/update.py

```diff
@@ -102,15 +107,19 @@ def update_content_graph(
                 )
                 return
     is_graph_up_to_date = content_graph_interface.import_graph(imported_path)
-    if not imported_path and not is_graph_up_to_date:
+    if not imported_path and not is_graph_up_to_date and not is_external_repository():
```

Comment:
for readability, I recommend `if not any((imported_path, is_graph_up_to_date, is_external_repository())`